docs/metrics/README.md

```diff
@@ -0,0 +1,94 @@
+# OpenTelemetry .NET Metrics
+
+## Best Practices
+
+- Instruments SHOULD only be created once and reused throughout the application
+  lifetime. This
+  [example](../../docs/metrics/getting-started-console/Program.cs) shows how an
+  instrument is created a `static` field and then used in the application. You
+  could also look at this ASP .NET Core
+  [example](../../examples/AspNetCore/Program.cs) which shows a more Dependency
+  Injection friendly way of doing this by extracting the `Meter` and an
```

Comment:
which shows a more Dependency Injection friendly way of doing this. -> I think this much is sufficient, to avoid too much details here.